_posts/2023-10-20-anvio-org-tutorial.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: blog
+authors: [ivagljiva]
+title: "How to add a post to anvio.org"
+excerpt: "You want to write a blog post or tutorial? We got you, fam."
+date: 2023-10-20
+tags: [contributors, technical]
+comments: true
+permalink: /docs/anvio-org-tutorial/
+---
+
+Anyone who is using anvi'o or has something to share with the anvi'o 'omics community should feel free to add their content to anvio.org. However, the process for doing that can be a bit confusing for newcomers. This tutorial will teach you how to contribute to this website, whether you want to write a blog post, a tutorial, or a technical write-up. It will use this very post that you are reading as an example to show you how. :)
```

Comment:
Suggestion here:

{:.notice}
The following are the steps to follow for someone who has the technical skills to send in a PR for new web material. But we don't want the anvi'o community to suffer from not hearing your voice if you are not someone who do not have the technical background to follow this tutorial. In that case, please get in touch with {% include person/display_mini_single.html github="meren" %}, who can help you with the technical part and turn your content into the necessary format.